htdocs/modulebuilder/index.php

```diff
@@ -711,7 +711,7 @@
 			$notnull = ($obj->Null == 'YES' ? 0 : 1);
 			if ($fieldname == 'fk_user_modif') $notnull = -1;
 			// label
-			$label = preg_replace('/_/', ' ', ucfirst($fieldname));
+			$label = $fieldname;
```

Comment:
I think the preg_replace was introduce to be compatible with old field name likes 'date_creation' to have a label that looks good for such cases.
In which cas did this annoy you ?

Reply:
This is annoying in the case I use $fieldname as key for translation as the space is not accepted in translation keys.

Reply:
So i suggest to remove _ instead of just replacing them with space.
Check this other fix is ok for you